ProfileService.js

```diff
@@ -34,16 +34,21 @@ export class ProfileService {
 
   /**
    *
-   * @param {Object} options - The options to use.
+   * @param {object} options - The options to use.
    * @param {string} [options.url = '/profiles'] - The service url to use.
    * @param {string} [options.account] - The account to associate with the
    *                                     profile agent.
+   * @param {object} [didMethod] - Supported: 'key' and 'v1'.
+   * @param {string} [didOptions] - Hashmap of optional DID method options.
    *
-   * @returns {Promise} Resolves when the operation completes.
+   * @returns {Promise<object>} Resolves when the operation completes.
    */
-  async create({url = this.config.urls.base, account} = {}) {
+  async create({
+    url = this.config.urls.base, account, didMethod, didOptions} = {}) {
```

Comment:
Two `did*` params seems a bit odd here.  Just bikeshedding here:
```
const didOptions = {
   method: 'key',
  ...
};
```

Reply:
@mattcollier I hear you; no real strong opinion here.

Fwiw, this is just matching the param naming downstream in `bedrock-profile` and `bedrock-profile-http`, that was suggested by Longley.

Reply:
@dmitrizagidulin OK, we've discussed this and for time purposes we'll go with this today and break it later if it seems important.